src/frontend/src/flows/about/copy.json

```diff
@@ -0,0 +1,24 @@
+{
+  "en": {
+    "header": "Securely connect to dapps on the Internet Computer",
+    "use_ii": "Use Internet Identity to manage your accounts on the IC without being tracked.",
+    "create_an_anchor": "Create an Identity Anchor",
+    "your_ii_is": "Your Identity Anchor is a number that connects you to your accounts on applications across the Internet Computer. Store your Anchor somewhere safe. If you lose it, you lose access to all of the accounts that you created with it. Internet Identity keeps your anchor private and does not share it with any applications.",
+    "easy_secure_login": "Easy and Secure Login",
+    "privacy_and_convenience": " Get the privacy of many separate accounts, with the convenience of a single sign-on. Internet Identity generates a new private key for every dapp that you use, and stores the keys on your device. With just your Identity Anchor and device, you can manage hundreds of private keys and accounts.",
+    "explore_dapps": "Explore dapps on the Internet Computer",
+    "ic_dapps": "With an Identity Anchor, you can create accounts with dapps on the Internet Computer, like:",
+
+    "distrikt_name": "Distrikt",
+    "distrikt_link": "https://distrikt.app/",
+    "distrikt_desc": "A decentralized, professional social media network that empowers users to own and control their identity and data.",
+
+    "discover_name": "Discovr",
+    "discover_link": "https://dscvr.ic0.app/",
```

Comment:
`dscvr.one`

Reply:
Same as above, there's a typo; will need to work with the relevant people to figure out why it was written like this

Reply:
Makes sense